csvqb/csvqb/cli/entrypoint.py

```diff
@@ -28,6 +28,17 @@ def entry_point():
     required=True,
     metavar="CONFIG_PATH",
 )
+@click.option(
+    "--catalog-metadata",
+    "-m",
+    help=(
+        "Location of a JSON file containing the Catalog Metadata for this qube. "
+        "If present, this overrides any configuration found in the info.json."
+    ),
+    type=click.Path(exists=True, path_type=Path, file_okay=True, dir_okay=False),
```

Comment:
This is one for the documentation on the series of overrides.

1) Catalog information from `info.json`
2) Catalog information from scraper
3) Catalog information from `catalog-metadata.json`
4) Runtime changes